source/pervasive/vec.rs

```diff
@@ -158,6 +158,13 @@ impl<A> Vec<A> {
     {
         self.vec.as_slice()
     }
+
+    #[verifier(external_body)]
+    pub fn clear(&mut self)
+        ensures self.len() == 0
```

Comment:
can you do `ensures self.view() == seq![]` instead? I know they're equivalent, but the more explicit version might prevent the user from having to call extensionality.

Reply:
Sure! -- I've used `self.view() == Seq::<A>::empty()`  as `seq![]` seems to have had problems with the type parameter.